Formula/halide.rb

```diff
@@ -36,18 +36,24 @@ class Halide < Formula
   depends_on "pybind11" => :build
   depends_on "jpeg-turbo"
   depends_on "libpng"
-  depends_on "python@3.10"
+  depends_on "python@3.11"
 
-  fails_with gcc: "5" # LLVM is built with Homebrew GCC
+  fails_with :gcc do
+    version "6"
+    cause "Requires C++17"
+  end
 
-  def install
-    args = %W[
-      -DCMAKE_INSTALL_RPATH=#{rpath}
-      -DHalide_SHARED_LLVM=ON
-      -DPYBIND11_USE_FETCHCONTENT=OFF
-    ]
+  def python3
+    "python3.11"
+  end
 
-    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
+  def install
+    system "cmake", "-S", ".", "-B", "build",
+                    "-DCMAKE_INSTALL_RPATH=#{rpath}",
+                    "-DHalide_INSTALL_PYTHONDIR=#{prefix/Language::Python.site_packages(python3)}",
```

Comment:
Existing bottle path: `halide/14.0.0_4/lib/python3/site-packages/halide.cpython-310-darwin.so`

Due to https://github.com/halide/Halide/blob/v14.0.0/packaging/CMakeLists.txt#L53-L54
```cmake
    set(Halide_INSTALL_PYTHONDIR "${CMAKE_INSTALL_LIBDIR}/python3/site-packages"
        CACHE STRING "Path to Halide Python bindings folder")
```